rules_building_block/discovery_net_share_discovery_winlog.toml

```diff
@@ -28,10 +28,10 @@ building_block_type = "default"
 query = '''
 sequence by user.name, source.port, source.ip with maxspan=15s 
  [file where event.action == "network-share-object-access-checked" and 
-  winlog.event_data.ShareName : ("\\*ADMIN$", "\\*C$") and 
+  winlog.event_data.ShareName in ("\\\\*\\ADMIN$", "\\\\*\\C$") and 
```

Comment:
```suggestion
  endswith~(winlog.event_data.ShareName, "ADMIN$", "C$") and 
```

Reply:
This would still match like a *C$

Reply:
updated the suggestion to exclude IPC$

Reply:
I mean, this would make the query more expensive and still match any computer account that ends with C$, the original suggestion seems to work just fine

![image](https://github.com/elastic/detection-rules/assets/26856693/e6f7fd86-f130-4d29-8184-8bcc247a429e)
![image](https://github.com/elastic/detection-rules/assets/26856693/5be39b78-73a6-4b6e-a8dd-d0d3db38af12)

Reply:
good catch, ingnore the above suggestion

Reply:
Awesome, thanks for the review